src/modules/search/components/SearchForm.tsx

```diff
@@ -37,6 +37,39 @@ export interface SearchFormProps {
 
 const defaultSearchKeys = ['textSearch', 'projects'];
 
+export const validateSearchFormInput = (values: Record<string, any>) => {
+  const requiredValueSet = omitBy(
+    pick(values, [
+      'personalId',
+      'firstName',
+      'lastName',
+      'ssnSerial',
+      'dob',
+      'textSearch',
+    ]),
+    (e) => isNil(e) || isEmpty(e)
+  );
+
+  const errors: Record<keyof typeof requiredValueSet, string> = {};
+
+  if (isEmpty(requiredValueSet)) {
+    errors.textSearch =
+      'Please enter a search term or specific search criteria';
```

Comment:
Can you put these strings in a translation? We're not using it consistently yet, but we are using it for other parts of the search form! (see `useTranslation` usage in this component)

Reply:
Fixed